crates/std_detect/src/detect/mod.rs

```diff
@@ -56,6 +56,12 @@ cfg_if! {
         mod aarch64;
         #[path = "os/freebsd/mod.rs"]
         mod os;
+    } else if #[cfg(all(target_os = "openbsd", target_arch = "aarch64", feature = "libc"))] {
+        #[allow(dead_code)]
```

Comment:
Why this?

Reply:
That module has code that calls the mrs instruction, but it is not used on OpenBSD.

Added a comment mentioning it.